assets/css/buttons.css

```diff
@@ -14,13 +14,13 @@
   font-weight: 400;
   font-style: normal;
   line-height: 1;
-  font-size: .75rem;
+  font-size: .75em;
```

Comment:
For sure. I'm just not sure what makes sense to scale up or down. I know the problem with pixels is that they don't scale based on browser settings. Some people who have poor eye-sight set their browsers to have a default font size that is larger, which doesn't change the scale for how large "pixels" are. That's the difference between the browser's zoom feature and the accessibility font-size feature. Everything scales with the zoom. The problem I'm seeing is for mobile users, where the zoom feature isn't an option. They can zoom in, but it's more like a magnifying glass. Some people don't know how to turn on the larger text accessibility feature (which is ironic). That makes me wonder if on mobile there should be an button that can make the text larger right in the page. Reasonably, that would only affect the font-size on the page, but maybe not the nav bar? Just an idea. Not a necessity though. We probably need to focus on the features that allow volunteers to actually use our site first though. I'll put in an issue for this, and then we can worry about it more later.